upgrade/upgrade.go

```diff
@@ -105,6 +106,9 @@ func upgrade(currentNodeID, namespace string, config *restclient.Config, lhClien
 				if err = doAPIVersionUpgrade(namespace, config, lhClient); err != nil {
 					return
 				}
+				if err = doPodsUpgrade(namespace, lhClient, kubeClient); err != nil {
```

Comment:
@shuo-wu @yasker does the order in which we upgrade the different components matter?
pods vs crds first?

Reply:
NO. It's fine as long as they can be updated successfully.